editor/src/layout/widgets.rs

```diff
@@ -170,7 +188,7 @@ pub enum Widget {
 #[derive(Clone, Serialize, Deserialize, Derivative)]
 #[derivative(Debug, PartialEq, Default)]
 pub struct NumberInput {
-	pub value: f64,
+	pub value: Option<f64>,
```

Comment:
Why should this be an option? We appear to just unwrap it in most cases anyway.

Reply:
`None` equates to a dash in the NumberInput GUI.